src/main/java/org/zalando/springframework/web/logging/DefaultHttpLogger.java

```diff
@@ -28,7 +28,7 @@
 import java.util.List;
 import java.util.Map;
 
-public class DefaultHttpLogger implements HttpLogger {
+public final class DefaultHttpLogger implements HttpLogger {
```

Comment:
You never know what crazy things clients of the library want to do, maybe they want to extend this class - so why prevent it?